ctk/common/src/main/kotlin/org/codice/compliance/verification/core/SamlDefinedIdentifiersVerifier.kt

```diff
@@ -64,7 +64,7 @@ internal class SamlDefinedIdentifiersVerifier(val node: Node) {
     /** 8 SAML-Defined Identifiers */
     fun verify() {
         verifyAttributeNameFormatIdentifiers()
-        verifyEmailAddressIdentifier()
+        // verifyEmailAddressIdentifier()
```

Comment:
✏️ Should leave a comment here indicating the reason for the commented out verification step and what it would require to re-enable it.